smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -156,7 +151,7 @@ open class HttpProtocolServiceClient(
     }
 
     private fun renderEmptyAsynchronousConfigInitializer(properties: List<ConfigProperty>) {
-        writer.openBlock("public convenience required init() async throws {", "}") {
+        writer.openBlock("public init() async throws {", "}") {
```

Comment:
In several places `convenience` and `required` are removed from config initializers since these modifiers are not used for value types.